projects/packages/my-jetpack/src/products/class-videopress.php

```diff
@@ -179,14 +179,22 @@ public static function get_manage_url() {
 	 * @return boolean
 	 */
 	public static function has_paid_plan_for_product() {
-		$purchases_data = Wpcom_Products::get_site_current_purchases();
+		$plans_with_videopress = array(
+			'jetpack_videopress',
+			'jetpack_complete',
+			'jetpack_business',
+			'jetpack_premium',
+		);
```

Comment:
Should Jetpack Security be included in here for VideoPress?
Oh wait, it looks to me like the free tier of VideoPress is included with Jetpack Security, but not the paid tier of VideoPress. Is that how you understand it? If that's the case, then this array is fine the way it is, right?

Reply:
Yes that is correct, here we are only adding the legacy plans to check for paid plans 😄